plugins/flusher/stdout/flusher_stdout.go

```diff
@@ -147,7 +144,11 @@ func (*FlusherStdout) IsReady(projectName string, logstoreName string, logstoreK
 }
 
 // Stop ...
-func (*FlusherStdout) Stop() error {
+func (p *FlusherStdout) Stop() error {
+	if p.outLogger != nil {
+		p.outLogger.Flush()
```

Comment:
// Close flushes all the messages in the logger and closes it. It cannot be used after this operation.
	Close()

	// Flush flushes all the messages in the logger.
	Flush()

--- Is no need to call Flush?

Reply:
yes,you are right. Test ok without call Flush.